src/main/java/ladder/domain/LadderLine.java

```diff
@@ -0,0 +1,30 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import ladder.domain.strategy.ConnectStrategy;
+
+public class LadderLine {
+
+  private final List<Boolean> isConnects;
+
+  private LadderLine(List<Boolean> isConnects) {
+    this.isConnects = isConnects;
+  }
+
+  public static LadderLine of(int userCount, ConnectStrategy connectStrategy) {
+    List<Boolean> connects = new ArrayList<>();
+    for (int i = 0; i < userCount; i++) {
+      connects.add(connectStrategy.isConnect(i));
+    }
+    return new LadderLine(connects);
```

Comment:
```suggestion
    return new LadderLine(connectStrategy.create());
```
제가 생각한 전략패턴을 사용했을 경우의 생성자 로직인데, 이렇게도 가능하지 않을까요?